src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+interface ButtonProps {
+	label: string;
+	callback: Function;
```

Comment:
This component would be a great start to use a [Shadcn](https://ui.shadcn.com/) here. 

Also, what I've been liking in the past for components is creating a folder like this:

`components/Button/index.ts` <- and using the `index.ts` to export the main component file as a barrel export. 

so essentially you'd have two files:`components/Button/Button.tsx` and `components/Button/index.ts`

Just let me know if that's cool with you? Or makes sense?

Reply:
I think I understand but I'd like to go over with it you to make sure I do